### ISSUE TITLE
dtype still float64 after `linear_scale_range` in geoparquet output

### ISSUE THREAD

**Issue by VincentVerelst** [Open-EO/openeo-geopyspark-driver#805]
I try to save my Sentinel-2 data as uint16 to a geoparquet file by using 
```python
...
cube = cube.linear_scale_range(0,65534,0,65534)
vector_cube = cube.aggregate_spatial(geometries, 'mean')
job = vector_cube.execute_batch(
    out_format='Parquet')
...
```
However, when inspecting the resulting geoparquet file, the datatype is still float64. A very basic process graph that replicates this result is j-240613692f534da4a97777026ec6b45f on CDSE

**Comment by jdries** [comment]
@VincentVerelst indeed, geoparquet does not support data types, and in fact we depend on the python numpy/xarray ecosystem here, which prefers to stick to floats, as that data type supports nodata.

Is there any urgent use case for having data types in vector cubes or parquet outputs?
